build.gradle

```diff
@@ -732,6 +742,10 @@ ext.libraries = [
             exclude(group: 'org.slf4j', module: 'slf4j-api')
             force = true
         },
+        spockframework   : [
```

Comment:
"Polluting" with Spock? No, it only makes the project tests "prettier" :-) Glad that from now on we could use Spock for testing :+1: